xnmt/batcher.py

```diff
@@ -112,7 +112,7 @@ def pack(self, src, trg):
     order = list(range(len(src)))
     return self.pack_by_order(src, trg, order)
 
-class ShuffleBatcher(Batcher):
+class ShuffleBatcher(Batcher, Serializable):
```

Comment:
Could you remove the Serializable here? It might trick people into thinking they can specify the ShuffleBatcher in the config file.